src/main/java/com/easypost/EasyPost.java

```diff
@@ -4,10 +4,9 @@
 
 public abstract class EasyPost {
     public static final String VERSION = "${project.version}";
-    public static String API_BASE = "https://api.easypost.com/v2";
-    public static String BETA_API_BASE = "https://api.easypost.com/beta";
+    public static final String API_BASE = "https://api.easypost.com/v2";
+    public static final String BETA_API_BASE = "https://api.easypost.com/beta";
     public static String apiKey;
-    public static String apiVersion;
```

Comment:
why is this removed?

Reply:
Wasn't being used anywhere. Might have been something that snuck in pre-emptively a while ago in anticipation of API versioning.